src/google/protobuf/stubs/stringpiece.h

```diff
@@ -391,7 +321,7 @@ class PROTOBUF_EXPORT StringPiece {
 // one of the arguments is a literal, the compiler can elide a lot of the
 // following comparisons.
 inline bool operator==(StringPiece x, StringPiece y) {
-  stringpiece_ssize_type len = x.size();
+  StringPiece::difference_type len = x.size();
   if (len != y.size()) {
```

Comment:
Some of the test runs are showing an error on this line:
```
../../../src/google/protobuf/stubs/stringpiece.h: In function ‘bool google::protobuf::stringpiece_internal::operator==(google::protobuf::stringpiece_internal::StringPiece, google::protobuf::stringpiece_internal::StringPiece)’:
../../../src/google/protobuf/stubs/stringpiece.h:325:11: error: comparison between signed and unsigned integer expressions [-Werror=sign-compare]
   if (len != y.size()) {
           ^
```
I think perhaps `len` should be a `size_type`.

Reply:
Changed to `size_type` (although I wasn't able to repro the error locally). PTAL, thanks!